tests/prettier/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -13,6 +13,54 @@ function fn() {
 "
 `;
 
+exports[`test jsx_long_name_closed.js 1`] = `
+"<BaseForm url=\"/auth/google\" method=\"GET\" colour=\"blue\" size=\"large\" submitLabel=\"Sign in with Google\"/>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<BaseForm
+  url=\"/auth/google\"
+  method=\"GET\"
+  colour=\"blue\"
+  size=\"large\"
+  submitLabel=\"Sign in with Google\"
+/>;
+"
+`;
+
+exports[`test jsx_long_name_open.js 1`] = `
+"<BaseForm url=\"/auth/google\" method=\"GET\" colour=\"blue\" size=\"large\" submitLabel=\"Sign in with Google\">
+  hello
+</BaseForm>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<BaseForm
+  url=\"/auth/google\"
+  method=\"GET\"
+  colour=\"blue\"
+  size=\"large\"
+  submitLabel=\"Sign in with Google\"
+>
```

Comment:
Most conventions I've seen is to put the `>` in the same line as the last attribute. This is likely going to be a bit trickier to implement but I think that it's important.